core/tauri-utils/src/acl/resolved.rs

```diff
@@ -307,6 +248,65 @@ fn parse_window_patterns(windows: HashSet<String>) -> Result<Vec<glob::Pattern>,
   Ok(patterns)
 }
 
+fn resolve_capability<F: FnMut(&str, &str, Commands, Scopes)>(
```

Comment:
the function name doesn't really convey what it does, should probably rename to `with_resolved_commands`?